README.rst

```diff
@@ -399,7 +399,7 @@ Contributing
 
    .. code-block:: text
 
-      python3 -m pip install -r requirements.txt
+      python3 setup.py install
```

Comment:
`setup.py install` is deprecated.

Reply:
where can i find the `requirements.txt`? On main I don't see one in the root dir, what am I missing?

Reply:
The readme file is a bit outdated. We use `tox` to run the tests, so the test dependencies are declared in `tox.ini`

Reply:
will adjust, thanks